packages/@coorpacademy-components/src/organism/list-item/index.js

```diff
@@ -48,19 +74,37 @@ const ListItem = ({
     ) : null;
 
   return (
-    <div className={classnames(style.wrapper, isBulkStyle && style.gridLayout)}>
+    <div
+      className={classnames(
+        style.wrapper,
+        isBulkStyle && style.gridLayout,
+        subtitle && style.withSubtitle,
+        disabled && style.disabled
+      )}
+      onClick={!disabled ? onClick : undefined}
+      style={selectedStyle}
+    >
       <div
         className={classnames(style.dataColumnsWrapper, isOverflowHidden && style.hiddenOverflowX)}
       >
         {isPublished && contentType === 'certification' ? orderView : null}
         <div className={style.title} title={title}>
           {title}
+          <div className={style.subtitle}>{subtitle}</div>
```

Comment:
conditional render ?
```suggestion
          {subtitle ? <div className={style.subtitle}>{subtitle}</div> : null}
```